Drivers/DataGenerators/DataGenerator.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Bogus;
+using Person;
+
+namespace Drivers;
+
+public class DataGenerator
+{
+    public static Driver CreateDriver()
+    {
+        var fakeDriver = new Faker<Driver>()
+            .RuleFor(p => p.Surname, f => f.Name.LastName())
+            .RuleFor(p => p.Name, f => f.Name.FirstName())
+            .RuleFor(p => p.DateOfBirth,
+                f => f.Date.Past(Driver.DriverMaxAge, DateTime.Now.AddYears(-Driver.DriverMinAge)))
+            .RuleFor(p => p.DateOfGetLicense, (f, p) => f.Date.Between(p.DateOfBirth.AddYears(16), DateTime.Now))
+            .RuleFor(p => p.LicenseId, f => f.Random.Guid());
+        Driver driver = fakeDriver.Generate();
+        driver.Vehicles = new List<Vehicle>();
+        for (int i = 0; i <= RandomUtils.GetDriverCarsAmount(); i++)
+        {
+            driver.Vehicles.Add(CreateVehicle());
+        }
```

Comment:
Remove this for loop. You can make method CreateVehicles() which will return List<Vehicle>